applications/tari_base_node/src/parser.rs

```diff
@@ -1517,10 +1520,9 @@ impl Parser {
         let msg = args.collect::<Vec<&str>>().join(" ");
 
         let fee_per_gram = 25 * uT;
-        let mut txn_service = self.wallet_transaction_service.clone();
-        self.executor.spawn(async move {
-            let event_stream = txn_service.get_event_stream_fused();
-            match txn_service
+        executor.spawn(async move {
+            let event_stream = wallet_transaction_service.get_event_stream_fused();
+            match wallet_transaction_service
```

Comment:
Clone `self.wallet_transaction_service` and move (i.e. pass ownership) the cloned version into the async block. A reference to `self` cannot be used with `executor.spawn` because the `spawn` requires a `'static` lifetime (i.e. all state that is used must either be owned or exist as long as the program execution).  

```suggestion
     let wallet_transaction_service = self.wallet_transaction_service.clone();

        self.executor.spawn(async move {
            let event_stream = wallet_transaction_service.get_event_stream_fused();
            match wallet_transaction_service
```

Reply:
As above